### PR TITLE
BAEL-2661 - Groovy def keyword

### PR DESCRIPTION
**Pull request by SmartyAnsh**

### CODE REVIEW

File: core-groovy-2/pom.xml [hunk 1]
```diff
@@ -43,13 +64,40 @@
...
+                            <goal>verify</goal>
+                        </goals>
+                        <configuration>
+                            <includes>
+                                <include>**/*Test5.java</include>
+                            </includes>
+                        </configuration>
+                    </execution>
+                </executions>
+			</plugin>
```
Comment (KevinGilmore) [inline core-groovy-2/pom.xml]: Let's use spaces for the indentation

File: core-groovy-2/pom.xml [hunk 2]
```diff
@@ -71,10 +119,16 @@
     </repositories>
 
     <properties>
+        <junit.platform.version>1.0.0</junit.platform.version>
+<!--         <groovy.version>2.4.13</groovy.version> -->
```
Comment (KevinGilmore) [inline core-groovy-2/pom.xml]: Let's remove these commented lines.

File: core-groovy-2/pom.xml (hunk 1 above)
Comment (SmartyAnsh) [reply core-groovy-2/pom.xml]: resolved

File: core-groovy-2/pom.xml (hunk 2 above)
Comment (SmartyAnsh) [reply core-groovy-2/pom.xml]: removed commented lines

**Pull request merged**